sql/2019/05_Third_Parties/05_01.sql

```diff
@@ -0,0 +1,23 @@
+#standardSQL
+# Percentage of pages that include at least one third party resource.
+SELECT
+  COUNT(*) as numberOfPages,
+  COUNTIF(numberOfThirdPartyRequests > 0) AS numberOfPagesWithThirdParty,
+  COUNT(*) / COUNTIF(numberOfThirdPartyRequests > 0) AS percentOfPagesWithThirdParty
```

Comment:
This needs to be flipped so the total is the denominator:

```suggestion
  COUNTIF(numberOfThirdPartyRequests > 0) / COUNT(*) AS percentOfPagesWithThirdParty
```